target/linux/realtek/image/rtl839x.mk

```diff
@@ -9,6 +9,15 @@ define Device/d-link_dgs-1210-52
 endef
 TARGET_DEVICES += d-link_dgs-1210-52
 
+define Device/netgear_gs750e
+  SOC := rtl8393
+  IMAGE_SIZE := 6528k
```

Comment:
I can confirm that it doesn't support lzma:
```
Loading ...

   Version:   MIPS OpenWrt Linux-5.10.161
   Created:   2023-01-05  16:17:24 UTC
   Size:      5157221 Bytes = 4.9 MB
   Checksum ... OK
Unimplemented compression type 3
```

However I was thinking that we can define a factory image using the max size of 6528k and a sysupgrade size of 7552k (the sum of the 3 concatenated devices). So that when the factory image won't fit anymore, we can still continue to use the sysupgrade image while the new installation can still be performed using the "tftpboot initramfs -> flash from OpenWrt" method:
```
# When the factory image won't fit anymore, it can be removed.
# New installation will be performed booting from initramfs
# and then flashing from OpenWrt
define Device/netgear_gs750e
  SOC := rtl8393
  IMAGE_SIZE := 7552k
  FACTORY_SIZE := 6528k
  DEVICE_VENDOR := NETGEAR
  DEVICE_MODEL := GS750E
  UIMAGE_MAGIC := 0x174e4741
  IMAGES += factory.bin
  IMAGE/factory.bin := \
    append-kernel | \
    pad-to 64k | \
    append-rootfs | \
    pad-rootfs | \
    check-size $$$$(FACTORY_SIZE)
endef
TARGET_DEVICES += netgear_gs750e
```
What do you think about it? I will also update the commit message accordingly

Reply:
For a factory image, you would need to create an initramfs with limited size. So if the initramfs image is also used for factory installs, then that could probably use a size check. (Meandering thought: maybe we should actually do that by default for all realtek devices...)

Reply:
Unfortunately the initramfs image is even bigger than the sysupgrade image... Probably because the sysupgrade has the squashfs partition that has a better compression than then the normal gzip? In the meantime I added the factory.bix image and updated the installation section of the commit message.